homeassistant/components/luftdaten/__init__.py

```diff
@@ -40,6 +40,10 @@
 SENSOR_PRESSURE_AT_SEALEVEL = "pressure_at_sealevel"
 SENSOR_TEMPERATURE = "temperature"
 
+SENSOR_LAEQ = "noise_LAeq"
```

Comment:
```suggestion
SENSOR_LA_EQ = "noise_la_eqeq"
```

Reply:
Here some example data:
```
    "sensor": {
        "id": 54519,
        "pin": "15",
        "sensor_type": {
            "id": 29,
            "name": "Laerm",
            "manufacturer": "Luftdaten.info"
        }
    },
    "sensordatavalues": [
        {
            "id": 6973654656,
            "value": "56.45",
            "value_type": "noise_LAeq"
        },
        {
            "id": 6973654734,
            "value": "45.14",
            "value_type": "noise_LA_min"
        },
        {
            "id": 6973654749,
            "value": "69.30",
            "value_type": "noise_LA_max"
        }
    ]
}
```

"value_type" is noise_LAeq, noise_LA_min and noise_LA_max as defined by luftdaten.info api. Not sure if noise_la_eqeq, noise_la_min and noise_LA_max will work.